RIPS/rip-7560.md

```diff
@@ -241,47 +242,54 @@ The amount of gas used by this frame is referred to as `senderValidationGasUsed`
 
 The frame must return 32 bytes `validationData` that is interpreted as:
 
-```solidity
-MAGIC_VALUE_SENDER <20 bytes>, validUntil <6 bytes>, validAfter <6 bytes>
-```
+- **authorizer** (20 bytes) - (magic value/address, defined below)
+- **validUntil** (6 bytes) - a timestamp, or zero for "infinite". The transaction is valid only up to this time.
+- **validAfter** (6 bytes) - a timestamp. The transaction is valid only after this time.
 
-In order to allow a gas estimation of the transaction prior signing, this function
-should avoid reverting on invalid signature, and return `MAGIC_VALUE_SIGFAIL` instead.
+Currently, **authorizer** is treated as a magic value:
+- `MAGIC_VALUE_SENDER` - returned by the account after it verified the transaction (and agrees to pay for its execution)
```

Comment:
Remove the brackets "(" and ")". They clutter the text and add no value.

Reply:
not sure to which brakets you mean to remove.